--- rustc_crates/rustc_target/src/abi/mod.rs	2022-01-30 21:22:38.390000000 -0500
+++ rustc_crates/rustc_target/src/abi/mod.new.rs	2022-01-30 23:27:16.120000000 -0500
@@ -168,7 +168,7 @@
             ));
         }
 
-        if dl.pointer_size.bits() != target.pointer_width.into() {
+        if dl.pointer_size.bits_usize() as u32 != target.pointer_width {
             return Err(format!(
                 "inconsistent target specification: \"data-layout\" claims \
                  pointers are {}-bit, while \"target-pointer-width\" is `{}`",
